native/packages/react-native-bpk-component-icon/stories.js

```diff
@@ -42,13 +48,96 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     flexWrap: 'wrap',
   },
+  column: {
+    flexDirection: 'column',
+    marginBottom: spacingBase,
+  },
+  singleRow: {
+    minWidth: 100,
+  },
+  singleIcon: {
+    paddingLeft: spacingSm,
+    paddingRight: spacingSm,
+  },
   icon: {
-    color: colorBlue700,
+    color: colorBlue500,
+  },
+  bottomMargin: {
+    marginBottom: spacingSm,
   },
 });
 
+/*
+ * the following functions should go on the bottom
+ * but eslint complains about it -_-
+ * @return
+ */
```

Comment:
> the following functions should go on the bottom but eslint complains about it -_-

Why should those functions go on the bottom?

Reply:
Otherwise i'd remove that comment :).

Reply:
oh good catch, well I was using regular functions before and es-lint was complaining about them being declared after the use, function hoisting is not a thing according to es-lint 😄